src/lib.rs

```diff
@@ -1 +1,118 @@
 #![allow(non_snake_case)]
+
+use std::ffi::c_void;
+use std::mem::size_of;
+use windows::Win32::Foundation::{CloseHandle, GetLastError, PWSTR};
+use windows::Win32::Security::SECURITY_ATTRIBUTES;
+use windows::Win32::System::Threading::{
+    TerminateProcess, WaitForSingleObject, PROCESS_CREATION_FLAGS, PROCESS_INFORMATION,
+    STARTUPINFOW,
+};
+use windows::Win32::System::WindowsProgramming::INFINITE;
+
+#[derive(Debug)]
+pub struct ChildProcess {
+    command: String,
+    process_information: PROCESS_INFORMATION,
+}
+
+pub type ChildProcessError = String;
```

Comment:
This is not an error type. Use the crate `thiserror` instead.

To be an error type you need your type to implement `std::error::Error`